docs/configuration.md

```diff
@@ -8,6 +8,7 @@ and variables needed to properly configure SC4S for your environment.
 | Variable | Values        | Description |
 |----------|---------------|-------------|
 | SC4S_USE_REVERSE_DNS | yes or no(default) | use reverse DNS to identify hosts when HOST is not valid in the syslog header |
```

Comment:
| yes or no (default) | Use reverse DNS to identify hosts when HOST is not valid in the syslog header. |